utils/serve.go

```diff
@@ -0,0 +1,26 @@
+package utils
+
+import (
+	"github.com/cheqd/did-resolver/types"
+	"github.com/rs/zerolog"
+	"github.com/rs/zerolog/log"
+)
```

Comment:
Is this necessary any more? See comment on above (since 'serve' is happening in main now)